docs/pages/docs/create-a-mintlist.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Create a Mintlist
+---
+
+# {% $markdoc.frontmatter.title %}
+
+You can create a Mintlist and populate it with configs for multiple NFTs. Users can interact with the Mintlist and
+mint NFTs from it one-by-one.
+
+To get started make sure you:
+
+## 1 - Download Glow
+
+If you don't have the Glow Solana wallet, you'll need to [download Glow](https://glow.app/download). Once you have that downloaded, you can
+
+## 2 - Get SOL
+
+You'll also need to have a small amount of SOL if you want to create an NFT on mainnet.
+
+You can buy SOL from Coinbase, Kraken, or with the "Buy" button inside of Glow.
+
+Now you're ready to go:
```

Comment:
I think we should include more information about mintlists. I don't expect people will do this before they create a single NFT so it doesn't make sense to just copy over those instructions.

Reply:
Changed the copy a bit and removed the first 2 steps